.github/workflows/torchbench.yml

```diff
@@ -97,4 +97,18 @@ jobs:
       exec_command: bash ./pytorch_blade/benchmark/TorchBench/test_torch_bench.sh cpu full
       push_command: ""
       remote_dev_docker: ""
-    secrets: inherit
\ No newline at end of file
+    secrets: inherit
+  TorchBenchCpuFull8Threads:
```

Comment:
better to encode CPU type into the name as well.

Reply:
right now cpu means x86 only, aarch64 has itw own name